tsconfig.json

```diff
@@ -10,6 +10,8 @@
     "module": "esnext",
     "moduleResolution": "node",
     "importHelpers": true,
+    "resolveJsonModule": true,
```

Comment:
What do you use it for?

Reply:
fake data.json with courses for now, I can remove it later

Reply:
Okay, got you.